core/services/log_zipper/main.py

```diff
@@ -20,10 +20,11 @@
 
 
 def zip_files(files: List[str], output_path: str) -> None:
-    with zipfile.ZipFile(output_path, "w", zipfile.ZIP_DEFLATED) as zipper:
+
         for file in files:
             logger.debug(f"Zipping {file}...")
-            zipper.write(file)
+            with open(file, 'rb') as f_in, gzip.open(file + ".gz", 'wb') as f_out:
+                f_out.writelines(f_in)
```

Comment:
Weird indentation

Reply:
why ?

Reply:
The error in CI is from wrong black version, I'll rebase over master.

Reply:
@joaoantoniocardoso check now